src/components/BasicDetailsForm.tsx

```diff
@@ -18,28 +32,62 @@ const useStyles = makeStyles((theme: Theme) =>
 	})
 );
 
-export default function BasicDetailsForm() {
-	const initialValues: IBasicInformation = {
-		confirmPassword: "",
-		email: "",
-		firstName: "",
-		lastName: "",
-		password: "",
-	};
+const BasicDetailsForm = () => {
+	const initialValues: IBasicInformation = getDefaultBasicInformation();
 	const classes = useStyles();
-	const navigate = useNavigate();
+	console.log(`styles`, classes);
 
-	function onSubmit(values: IBasicInformation, formikHelpers: FormikHelpers<IBasicInformation>) {
+	// console.log(`initialValues`, initialValues);
+
+	const { error, loading, data: singupSuccessfulResponse, setPayload } = useSignupNewUser();
```

Comment:
`useSignupNewUser` can be simplified to a generic `usePost` hook.
The custom hook should only be made to use similar state-dependent logic in multiple components.
currently `useSignupNewUser` can only be used to signUp a user.